src/main/java/core/basesyntax/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
```

Comment:
I personally do not see any sense to add Service word into name of class when there is a noun that describes action (like Reader, Writer, Processor, Handler etc.)
```suggestion
public class FileReaderImpl implements FileReader {
```